ledger-core/v2/application/functest/wallet_create_test.go

```diff
@@ -15,14 +15,14 @@ import (
 )
 
 // Creates wallets and check Reference format in response body.
-func TestCreateWallet(t *testing.T) {
+func TestWalletCreate(t *testing.T) {
 	status := getStatus(t)
 	require.NotEqual(t, 0, status.WorkingListSize, "not enough nodes to run test")
 	count := 50 * status.WorkingListSize // 250 for 5 nodes
```

Comment:
What information does this comment give me?

Reply:
how many requests to /wallet/create will be

Reply:
why do I need it?=) as a code reader?

Reply:
removed